### PR TITLE
migrate to null safety

### PR DESCRIPTION
**Pull request by Ali1Ammar**
migrate to null safety

### CODE REVIEW

File: lib/src/bouncing_grid.dart [hunk 1]
```diff
@@ -53,22 +53,14 @@ class LoadingBouncingGrid extends StatefulWidget {
     this.inverted = false,
     this.itemBuilder,
     this.duration = const Duration(milliseconds: 1500),
-  })  : assert(backgroundColor != null,
-            'loading_animations: property [backgroundColor] must not be null. Prefer using Colors.transparent instead.'),
```
Comment (cytryn) [inline lib/src/bouncing_grid.dart:57]: why did you remove the developer warnings?

Comment (Ali1Ammar) [reply lib/src/bouncing_grid.dart:57]: its non nullable now 
developer can't set null to it so condition won't be false at all

Comment (cytryn) [reply lib/src/bouncing_grid.dart:57]: lol right, that makes sense 🤣

**Review by cytryn — APPROVED** [review]
🚀
